packages/shared/src/variants/super_tic_tac_go.ts

```diff
@@ -1,159 +1,179 @@
+    return new Nonant(coord.x % 3, coord.y % 3);
+  }
 }
 
 export class SuperTicTacGo extends Baduk {
-    constructor(config: BadukConfig) {
-        super(SuperTicTacGo.sanitizeConfig(config));
-    }
+  constructor(config: BadukConfig) {
+    super(SuperTicTacGo.sanitizeConfig(config));
+  }
 
-    override playMove(player: number, move: string): void {
-        // move.length === 2 is checking to make sure it's not a 'special' move
-        // i should use a better move type checker for this
-        if (move.length === 2 && this.last_move.length === 2) {
-            const coordinate = super.decodeMove(move);
-            const targetNonant = this.getLegalNonant();
-            if (
-                !targetNonant.has(coordinate) 
-                && this.hasOpenPoints(targetNonant)
-            ) {
-                throw new Error (`${move} out of range of legal nonant`);
-            }
-        }
-    
-        super.playMove(player, move);
+  override playMove(player: number, move: string): void {
+    // move.length === 2 is checking to make sure it's not a 'special' move
+    // i should use a better move type checker for this
+    if (move.length === 2 && this.last_move.length === 2) {
+      const coordinate = super.decodeMove(move);
+      const targetNonant = this.getLegalNonant();
+      if (
+        !targetNonant.has(coordinate) &&
+        this.hasOpenPoints(targetNonant)
+        // (playing out of legal nonant
+        // is allowed if it has no open points)
+      ) {
+        throw new Error(`${move} out of range of legal nonant`);
+      }
     }
 
-    
-    override postValidateMove(): void {
-        // Allow self-capture moves
-        // allow  ko
-    }
+    super.playMove(player, move);
+  }
 
-    override prepareForNextMove(move: string): void {
-        if (move === 'pass') this.captures[this.next_to_play > 0 ? 0 : 1];
-        super.prepareForNextMove(move);
-    }
+  override postValidateMove(): void {
+    // Allow self-capture moves
+    // allow  ko
+  }
 
-    protected getLegalNonant(): Nonant {
-        const lastMove = super.decodeMove(this.last_move);
-        return Nonant.correspondingToCoordinate(lastMove);
-    }
+  override prepareForNextMove(move: string): void {
+    if (move === "pass") this.captures[this.next_to_play > 0 ? 0 : 1]++;
+    super.prepareForNextMove(move);
+  }
 
-    protected getOpenVerticesInNonant(nonant: Nonant): Array<Coordinate> {
-        return nonant.vertices().filter((coord) => {
-            return this.board.at(coord) === Color.EMPTY;
-        });
+  public getLegalNonant(): Nonant {
+    if (this.last_move.length !== 2) {
+      throw new Error("all nonants are legal");
     }
+    const lastMove = super.decodeMove(this.last_move);
+    return Nonant.correspondingToCoordinate(lastMove);
+  }
 
-    protected hasOpenPoints(nonant: Nonant): boolean {
-        return this.getOpenVerticesInNonant(nonant).length > 0;
+  // mostly added this for easier testing
```

Comment:
tbh, I think the test is fine without exposing this! I left another comment on the test about encapsulation